hdfs/client.py

```diff
@@ -602,6 +602,31 @@ def rename(self, hdfs_src_path, hdfs_dst_path):
         self.resolve(hdfs_src_path), hdfs_dst_path
       )
 
+  def change_owner(self, hdfs_path, owner, group=None):
+    """Change the owner of file.
+
+    :param hdfs_path: HDFS path.
+    :param owner: New owner for file.
+    :param group: Optional, new group for file.
+    """
+    if group is None:
+      res = self._set_owner(hdfs_path, owner=owner)
+    else:
+      res = self._set_owner(hdfs_path, owner=owner, group=group)
+    if not res.json()['boolean']:
```

Comment:
Could you check that the response contains a JSON object? I believe it contains an empty body.